### PR TITLE
InspectBgp added

### PR DESCRIPTION
**Pull request by melanietom**
InspectBgp is added to ofnet

### CODE REVIEW

File: ofnet.go [hunk 1]
```diff
@@ -89,6 +89,9 @@ type OfnetDatapath interface {
 
 	// Return the datapath state
 	InspectState() (interface{}, error)
+
+	//Inspect bgp
+	InspectBgp() (interface{}, error)
```
Comment (abhinandanpb) [inline ofnet.go]: can you rename this to InspectProto ?

File: ofnetAgent.go [hunk 2]
```diff
@@ -919,6 +919,13 @@ func (self *OfnetAgent) GetEndpointStats() ([]*OfnetEndpointStats, error) {
 	return self.datapath.GetEndpointStats()
 }
 
+// InspectBgp returns ofnet bgp state
+func (self *OfnetAgent) InspectBgp() (interface{}, error) {
+	log.Infof("OfnetAgent. InspectBgp")
+	peer, err := self.protopath.InspectBgp()
```
Comment (abhinandanpb) [inline ofnetAgent.go]: This would eventually call InspectProto

File: ofnetBgp.go [hunk 3]
```diff
@@ -803,3 +808,37 @@ func (self *OfnetBgp) sendArpPacketOut() {
 	// Send it out
 	self.agent.ofSwitch.Send(pktOut)
 }
+
+func (self *OfnetBgp) InspectBgp() (interface{}, error) {
```
Comment (abhinandanpb) [inline ofnetBgp.go]: This would be InspectProto

File: ofnetAgent.go [hunk 4]
```diff
@@ -919,6 +919,13 @@ func (self *OfnetAgent) GetEndpointStats() ([]*OfnetEndpointStats, error) {
 	return self.datapath.GetEndpointStats()
 }
 
+// InspectBgp returns ofnet bgp state
+func (self *OfnetAgent) InspectBgp() (interface{}, error) {
+	log.Infof("OfnetAgent. InspectBgp")
```
Comment (abhinandanpb) [inline ofnetAgent.go]: you can remove this comment

File: ofnetBgp.go [hunk 5]
```diff
@@ -803,3 +808,37 @@ func (self *OfnetBgp) sendArpPacketOut() {
...
+	log.Infof("OfnetBgp. InspectBgp")
```
Comment (abhinandanpb) [inline ofnetBgp.go]: you can remove this log as well

File: ofnetBgp.go [hunk 6]
```diff
@@ -803,3 +808,37 @@ func (self *OfnetBgp) sendArpPacketOut() {
...
+
+	// Get rib info
+	table := &api.Table{Type: api.Resource_GLOBAL, Family: uint32(bgp.RF_IPv4_UC)}
+	OfnetBgpInspect.Rib, err = client.GetRib(context.Background(), table)
+	if err != nil {
+		log.Errorf("GetRib failed: %v", err)
+		return nil, err
+	}
+
+	log.Infof("Peer: %v\n", OfnetBgpInspect.Peer)
```
Comment (abhinandanpb) [inline ofnetBgp.go]: all these logs can be removed as well .

**Comment by abhinandanpb** [comment]
changes LGTM. except minor comments. Thanks for taking care of this. It was long pending.

**Comment by shaleman** [comment]
sanity failed because of gofmt
```
./checks "./*.go ./ofctrl/ ./ovsdbDriver/ ./pqueue/ ./rpcHub/"
+++ Checking gofmt...
!!! gofmt checking failed:
./ofnetBgp.go

Makefile:22: recipe for target 'host-build' failed
make: *** [host-build] Error 1
make: *** [build] Error 2
Build step 'Execute shell' marked build as failure
```

can you please run gofmt on the changes?

**Pull request merged**